core/services/network_checker.rb

```diff
@@ -23,9 +23,14 @@ def self.resources_available?(machine_configurator, machine, logger)
            else
              :wget
            end
-    result = random_resources(resources).all? do |resource|
-      check_resource?(tool, machine_configurator, machine, resource, logger)
+    result = true
+    table = ['Network resource availability check:']
+    resources.each do |resource|
```

Comment:
Rewrite the implementation in the following way:
1. Convert a list of resources to an array of hashes that contain the name of the resource and it's availability status.
2. Print a table of availability status.
3. Make a judgement whether all resources are available or not.